test/fixtures/referrals.yml

```diff
@@ -0,0 +1,9 @@
+one:
```

Comment:
I added these fixtures so the "referred by" text will show up in the mailer previews.

Reply:
Hmm, shouldn't they appear in the database seeds then?

Reply:
I guess I misunderstood that. I removed the fixtures.